ooni/nettests/experimental/tls_handshake.py

```diff
@@ -743,6 +766,7 @@ def handshakeSucceeded(connection):
 
             self.report['host'] = host
             self.report['port'] = port
+            self.report['sni'] = self.sni
```

Comment:
I was unsure if there was some reason to bind `host` and `port` so late, so I just droped `sni` flag close to the host/port pair. Are you aware of any reason to do that?